packages/nativescript-appdynamics/platforms/android/include.gradle

```diff
@@ -0,0 +1,32 @@
+
+android {
+  defaultConfig {
+    minSdkVersion 30
+    generatedDensities = []
+  }
+  aaptOptions {
+    additionalParameters "--no-version-vectors"
+  }
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
+}
+
+apply plugin: "adeum"
+
+//optional elements
+dependencies {
+ implementation 'com.appdynamics:appdynamics-gradle-plugin:22.2.2' // this line added for AppDynamics
+ implementation 'com.appdynamics:appdynamics-runtime:22.2.2' // this line added for AppDynamics
+}
+adeum { // this section added for AppDynamics
+  account {
+
+    name 'JesperRellmeSEaccount-a0Q3400000DHE1FEAX'
+    licenseKey '401a2890-76d0-4cc5-876f-3b8b82dc4fec'
```

Comment:
This seems undesireable

Reply:
I've used a rebase to remove it from the file & history; I'm not sure why there's a license key in a build script like this.